docs/source/quantization.rst

```diff
@@ -38,20 +38,177 @@ that perform all or part of the computation in lower precision. Higher-level
 APIs are provided that incorporate typical workflows of converting FP32 model
 to lower precision with minimal accuracy loss.
 
-Quantization requires users to be aware of three concepts:
+Quantization Stack
```

Comment:
would it make sense to put the information for the common user (don't care about internals, just want to know how to quantize something with default settings) at the top, and put the details at the bottom?  Seems like this is currently doing the reverse.